templates/people/edit.html

```diff
@@ -12,5 +12,12 @@ <h1>{{ user }}{% if role %} <small>{{ role }}</small>{% endif %}</h1>
 		{{ form | crispy }}
 		<input type="submit" class="btn btn-primary" value="Save profile" />
 	</form>
+
+	<br>
+	<form method="POST" action="{% url 'people:profile_delete' fair.year user.pk %}">
+	{% csrf_token %}
+	<input type="submit" class="btn btn-primary" value="Delete profile" onclick="return confirm('Are you sure that you want to delete your profile? This means that all your data will be erased from Armada and cannot be restored. This means that if you want to use Armada in the future, you will have to create a new profile.')" />
```

Comment:
If you change `btn-primary` to `btn-danger` then it will give us a red button.
![image](https://user-images.githubusercontent.com/25460850/141331177-d79831f0-4a67-457c-b313-fc63c47c51c0.png)
[Maybe you can find buttons that works here](https://getbootstrap.com/docs/4.0/components/buttons/)

Reply:
@moaengquist @rrudling What do you think? Should it be grey outlined instead of a color?

Reply:
I will change it to red, thanks!